src/docs/product/alerts-notifications/notifications/index.mdx

```diff
@@ -50,7 +50,10 @@ Deploy emails are sent to users who have committed to the release that was deplo
 
 Sentry sends all owners of the organization emails when:
 
-- The event volume is approaching or has exceeded the quota
+- 80% of the error volume has depleted
+- Any data type (errors or events) has exceeded the quota
```

Comment:
@maheskett does this include transaction data?

Reply:
Yes, I think this sounds reasonable:

```
- 80% of the error volume has been depleted
- Errors or transactions have exceeded quota
```

Also, the notion doc is for our own use as many of these notifications are in flux, so I'd just use your best judgement about what makes sense to share in the public docs.

Reply:
@adhiraj are you okay with these recent updates?

Reply:
+1 to megan's suggestion "Errors or transactions have exceeded their quota"